net-libs/libwebsockets/metadata.xml

```diff
@@ -19,12 +19,15 @@
 		libwebsockets is a C library designed to make development of websockets clients and servers easier (possible).
 	</longdescription>
 	<use>
-		<flag name="ssl">Include SSL support (default <pkg>dev-libs/openssl</pkg>)</flag>
-		<flag name="zlib">Include zlib support</flag>
-		<flag name="static-libs">Build the static version of the library</flag>
+		<flag name="client">The client part of the library</flag>
 		<flag name="http2">Compile with support for http2</flag>
 		<flag name="ipv6">Compile with support for ipv6</flag>
```

Comment:
$ quse -D ipv6
 global:ipv6: Add support for IP version 6
 local:ipv6:net-libs/libwebsockets: Compile with support for ipv6

Looks like the metadata entry is redundant and can be safely removed.